controllers/multiclusterobservability/observatorium.go

```diff
@@ -220,6 +224,16 @@ func newDefaultObservatoriumSpec(mco *mcov1beta2.MultiClusterObservability,
 	return obs
 }
 
+// return proxy variables
+// OLM set these environment variables as a unit
+func newEnvVars() map[string]string {
+	return map[string]string{
```

Comment:
do we need to check one env exists or not before use it?

Reply:
OLM handles these environment variables as a unit. we do not need to check one exists or not.